sparkprs/controllers/tasks.py

```diff
@@ -51,6 +51,21 @@ def backfill_prs():
     return "Enqueued tasks to backfill %i PRs" % latest_pr_number
 
 
+@tasks.route("/github/clean-prs")
+def clean_deleted_prs():
+    prs = Issue.query(Issue.state == "open")
+    pr_list = ""
+    for pr in prs:
+        try:
+            raw_github_request(get_pulls_base() + '/%i' % pr.number,
+                               oauth_token=oauth_token, etag=pr.etag)
+        except:
```

Comment:
do we want to check for a specific error code ? Say 404 instead of a 503 (which we may get when github is down ?)